.prettierrc

```diff
@@ -1,3 +1,4 @@
 {
-  "printWidth": 80
+  "printWidth": 80,
+  "endOfLine":"auto"
```

Comment:
please remove this change

Reply:
Hi, It's done. I made this change because when I run the command "npm run lint" I had a lot of errors 
![image](https://user-images.githubusercontent.com/64835399/193217893-db581c30-073c-4f07-a58e-98f1252d81fe.png).

I also re-generate the package-lock.json.